stock_picking_eco_tag/report/stock_picking_eco_tag_template.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="label_transfer_template_eco_view_pdf">
+        <t t-call="web.basic_layout">
+            <div class="page">
+                <t t-foreach="docs" t-as="picking">
+                    <t t-foreach="picking.move_lines" t-as="move">
+                        <t t-foreach="move.move_line_ids" t-as="move_line">
+                          <div style="display: inline-table; height: 160px; width: 367px; border: 1px solid black;">
+                            <div class="d-flex">
+                                <div style="display: -webkit-box; display: -webkit-flex; display: flex; -webkit-box-align: center; justify-content: center; align-items: center; margin-left: 3px; width: 35%">
+                                    <img t-attf-src="data:image/jpg;base64,{{ move.company_id.logo }}" style="width: 130px"/>
+                                </div>
+                                <div style="width: 65%; padding-left: 6px;" t-if="picking.move_lines">
+                                    <div style="padding-top: 5px;">
+                                        <p class="mb-0" style="font-size: 10px">Product: <t t-esc="move_line.product_id.name"/></p>
+                                        <p class="mb-0" style="font-size: 10px">Var / Cat: <t t-esc="move_line.product_id.pos_categ_id.name"/></p>
```

Comment:
Wrong reference to a field (`pos_categ_id`) not linked in addon dependencies, if point_of_sale is not installed an error is fired. Please try `categ_id` instead.